check-ssl-expiry/index.js

```diff
@@ -0,0 +1,34 @@
+const https = require("https");
+
+// List of domains to check
+const domains = ["example.com", "google.com", "facebook.com"];
```

Comment:
Ideally use a csv + csv file reader! Thinking we can hardcode the csv value from the redirections directly, then we can import it as is into our backend

Reply:
good point, will add in future PRs